tests/conftest.py

```diff
@@ -760,7 +760,7 @@ def map_data_line(line):
 
 
 @pytest.fixture(scope="function")
-async def server_config(event_loop, inmanta_config, postgres_db, database_name, clean_reset, unused_tcp_port_factory):
+async def server_config(inmanta_config, postgres_db, database_name, clean_reset, unused_tcp_port_factory):
```

Comment:
I got warnings that this fixture should not be used with async test cases so I dropped it where I found it.